lib/adapters/outline-view-adapter.ts

```diff
@@ -42,15 +43,59 @@ export default class OutlineViewAdapter {
     const results = await Utils.doWithCancellationToken(connection, this._cancellationTokens, (cancellationToken) =>
       connection.documentSymbol({textDocument: Convert.editorToTextDocumentIdentifier(editor)}, cancellationToken),
     );
-    results.sort(
-      (a, b) =>
-        (a.location.range.start.line === b.location.range.start.line
-          ? a.location.range.start.character - b.location.range.start.character
-          : a.location.range.start.line - b.location.range.start.line),
-    );
-    return {
-      outlineTrees: OutlineViewAdapter.createOutlineTrees(results),
-    };
+
+    if ((results.length !== 0) &&
```

Comment:
Did you mean for results.length === 0 to go through the createOutlineTrees path in the else below?